Assets/BossRoom/Scripts/Server/Game/Entity/ServerProjectileLogic.cs

```diff
@@ -82,6 +82,11 @@ public override void OnNetworkSpawn()
             m_NpcLayer = LayerMask.NameToLayer("NPCs");
         }
 
+        public override void OnNetworkDespawn()
+        {
+            m_Started = false;
```

Comment:
Why not just use "IsSpawned"?

Reply:
Performance. These types of Netcode API calls on FixedUpdates really accumulate on lower-end devices.

Reply:
oh because of NetworkBehaviour's get on NetworkObject which is doing a bunch of things? right. This is dumb, we should really have a way to do something like Get NetworkObjectUnchecked, something like that. Anyway, ok approving this PR